### PR TITLE
Dev: Fix dashboard cndt2022-onsite

### PR DESCRIPTION
**Pull request by t4i5uKE**
# Description

Fixes # (issue)

## Type of change

- [ ] Document update or simple typo fix
- [ ] Maintenance/update components
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] I have checked backward/forward compatibility that may cause regarding this change.

### CODE REVIEW

**Review by kkmnky — APPROVED** [review]
LGTM

File: dashboards/cndt2022-onsite.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
       },
       "id": 8,
       "options": {
-        "content": "# Onsiteダッシュボードの使い方\n\n`オンサイト関連のメトリクス`を表示するダッシュボードです。\nCNDT2022からハイブリット開催ということで実験的にTrack Aの会場にカメラやセンサを設置して会場の人数やCO2濃度を取得しました。\n今回の実験を受けて次回から全会議室の情報を取得するなど更新していきたいと考えています。\n\n現在表示しているメトリクスは下記の通りです。\n\n- 人数: ※設置場所によって書き換える\n- CO2濃度: CO2センサによって出力されたCO2濃度を表示しています。",
+        "content": "# Onsiteダッシュボードの使い方\n\n`オンサイト関連のメトリクス`を表示するダッシュボードです。\nCNDT2022からハイブリット開催ということで実験的に会場のRoom1にカメラやセンサを設置して会場の人数やCO2濃度を取得しました。\n今回の実験を受けて次回から全会議室の情報を取得するなど更新していきたいと考えています。\n\n現在表示しているメトリクスは下記の通りです。\n\n- 人数: 2台のカメラから画像認識によって検出された人数を表示しています\n- CO2濃度: CO2センサによって出力されたCO2濃度を表示しています。",
```
Comment (faruryo) [inline dashboards/cndt2022-onsite.json:45]: `ハイブリット開催`→`ハイブリッド開催`

Comment (faruryo) [reply dashboards/cndt2022-onsite.json:45]: Onsiteとオンサイトはどちらかに揃えた方が良さそうです

Comment (inductor) [inline dashboards/cndt2022-onsite.json:45]:
```suggestion
        "content": "# オンサイトダッシュボードの使い方\n\n`オンサイト関連のメトリクス`を表示するダッシュボードです。\nCNDT2022からハイブリッド開催ということで実験的に会場のRoom1にカメラやセンサを設置して会場の人数やCO2濃度を取得しました。\n今回の実験を受けて次回から全会議室の情報を取得するなど更新していきたいと考えています。\n\n現在表示しているメトリクスは下記の通りです。\n\n- 人数: 2台のカメラから画像認識によって検出された人数を表示しています\n- CO2濃度: CO2センサによって出力されたCO2濃度を表示しています。",
```

**Review by inductor — CHANGES_REQUESTED** [review]

**Review by faruryo — APPROVED** [review]

**Pull request merged**